{"name": "针对异常登陆的测试", "status": "passed", "steps": [{"name": "步骤1：登录", "status": "passed", "start": 1610940143101, "stop": 1610940143103}], "attachments": [{"name": "log", "source": "c13e5c5a-219a-4f3c-9393-e15c24a8a860-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "data", "value": "{'name': '登录功能-异常登录-用户名为空', 'username': '', 'password': '654321'}"}], "start": 1610940143095, "stop": 1610940144364, "uuid": "72eec05b-8a3a-489e-9056-b1ce50000872", "historyId": "d93c8b521d1ee7fc4eb27338522fd359", "testCaseId": "130e6dce60bc4ef9d882c5f0465dc1d6", "fullName": "cases.test_login.TestLogin#test_login_failed", "labels": [{"name": "feature", "value": "金融界web登录模块"}, {"name": "story", "value": "登录模块-异常登陆"}, {"name": "epic", "value": "金融界web自动化自测"}, {"name": "tag", "value": "@pytest.mark.usefixtures('refresh_page')"}, {"name": "parentSuite", "value": "cases"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "PC-20200718ELAI"}, {"name": "thread", "value": "10944-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "cases.test_login"}]}